nav2_planner/src/planner_server.cpp

```diff
@@ -446,46 +446,47 @@ void PlannerServer::computePlanThroughPoses()
 
     action_server_poses_->succeeded_current(result);
   } catch (nav2_core::InvalidPlanner & ex) {
-    exceptionWarning(curr_start, curr_goal, goal->planner_id, ex);
+    exceptionWarning(curr_start, curr_goal, goal->planner_id, ex, result->error_msg);
```

Comment:
Where is `error_msg` populated in planner server?

Reply:
result->error_msg is sent in to exceptionWarning to grab the same message that is logged with RCLCPP_WARN.  IOW result->error_msg is an output of exceptionWarning.